x-pack/plugins/ml/server/routes/anomaly_detectors.ts

```diff
@@ -315,8 +310,6 @@ export function jobRoutes({ router, mlLicense }: RouteInitialization) {
    * @api {post} /api/ml/anomaly_detectors/_validate/detector Validate detector
    * @apiName ValidateAnomalyDetector
    * @apiDescription Validates specified detector.
```

Comment:
This needs an `@apiSchema` param.

Reply:
indeed, but it has not been described yet (`schema.any()`)